src/protocol/java/src/main/java/org/apache/jmeter/protocol/java/sampler/JavaSampler.java

```diff
@@ -121,9 +121,9 @@ private void initClass() {
             javaClass = Class.forName(name, false, Thread.currentThread().getContextClassLoader());
             Method method = javaClass.getMethod("teardownTest", JavaSamplerContext.class);
             isToBeRegistered = !method.getDeclaringClass().equals(AbstractJavaSamplerClient.class);
-            log.info("Created class: {}. Uses tearDownTest: ", name, isToBeRegistered);
+            log.info("Created class: {}. Uses tearDownTest: {}", name, isToBeRegistered);
         } catch (Exception e) {
-            log.error("{}\tException initialising: ", whoAmI(), name, e);
+            log.error("{}\tException initialising: {}", whoAmI(), name, e);
```

Comment:
While the text in the message looks like it is missing a placeholder, I would refrain from adding it here. Adding a placeholder will loose the stacktrace, which could be helpful identifying the root cause.